### PR TITLE
chore: add `round_timeout_seconds` to agent

### PR DESCRIPTION
**Pull request by 0xArdi**

### CODE REVIEW

File: packages/valory/agents/mech/aea-config.yaml [hunk 1]
```diff
@@ -133,14 +133,15 @@ models:
...
+      round_timeout_seconds: ${int:30.0}
       reset_period_count: ${int:1000}
       on_chain_service_id: ${int:1}
       multisend_address: ${str:0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761}
       service_registry_address: ${str:0x9338b5153AE39BB89f50468E608eD9d764B755fD}
       setup:
         all_participants: ${list:["0x10E867Ac2Fb0Aa156ca81eF440a5cdf373bE1AaC"]}
         safe_contract_address: ${str:0x5e1D1eb61E1164D5a50b28C575dA73A29595dFf7}
-      file_hash_to_tools_json: ${list:[["bafybeihhxncljjtzniecvm7yr7u44g6ooquzqek473ma5fcnn2f6244v3e",
+      file_hash_to_tools_json: ${list:[["bafybeib6ro4lzsm4jk43bzgqzxyoiabyhlcx5ejdd5z6rrfffktbnhlerm",
```
Comment (DavidMinarsch) [inline packages/valory/agents/mech/aea-config.yaml:144]: You didn't commit any changes to that file?

File: packages/valory/services/mech/service.yaml [hunk 2]
```diff
@@ -158,7 +158,7 @@ type: skill
         reset_pause_duration: ${RESET_PAUSE_DURATION:int:10}
         request_retry_delay: 1.0
         request_timeout: 10.0
-        round_timeout_seconds: 30.0
+        round_timeout_seconds: ${ROUND_TIMEOUT:int:150.0}
```
Comment (DavidMinarsch) [inline packages/valory/services/mech/service.yaml:161]: Applying to all skills could cause a problem. So let's open an issue for a longer term fix

File: packages/valory/agents/mech/aea-config.yaml (hunk 1 above)
Comment (0xArdi) [reply packages/valory/agents/mech/aea-config.yaml:144]: its not versioned, its on ipfs

File: packages/valory/services/mech/service.yaml (hunk 2 above)
Comment (0xArdi) [reply packages/valory/services/mech/service.yaml:161]: https://github.com/valory-xyz/open-autonomy/issues/1983

File: packages/valory/agents/mech/aea-config.yaml (hunk 1 above)
Comment (DavidMinarsch) [reply packages/valory/agents/mech/aea-config.yaml:144]: It should be committed in the same repo (in the `tools.py` folder)

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**